Tenduke.Client.DefaultBrowser/Authorization/AuthorizationCodeGrant.cs

```diff
@@ -19,9 +19,7 @@ namespace Tenduke.Client.DefaultBrowser.Authorization
     /// <summary>
     /// Authorization Code Grant flow using OS default browser.
     /// </summary>
-    [Serializable]
-    public class AuthorizationCodeGrant : Authorization<
-        AuthorizationCodeGrant, IDefaultBrowserAuthorizationCodeGrantConfig, AuthorizationCodeGrantArgs>
```

Comment:
The type parameter `T` of the `Authorization` generic type was not referenced in the code.

If code needs to know the type of the object, `IsAssignableFrom` of pattern matching will be easier to read.